rust/src/lib/route.rs

```diff
@@ -78,6 +78,22 @@ impl Routes {
                     error!("{}", e);
                     return Err(e);
                 }
+                if !route.destination.as_deref().unwrap_or_default().is_empty() {
```

Comment:
Please use `if let Some(dst) = route.destination.as_ref()` to save the extra check on `String::new().is_empty()`.

Reply:
I updated. Thanks.